<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game</class>
 <widget class="QMainWindow" name="Game">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">Game #Talon{
	border: 1px solid red;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Player1 : tempCardPosition : </string>
    </property>
   </widget>
   <widget class="QLabel" name="onMoveLabel">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>40</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>on move : player1</string>
    </property>
   </widget>
   <widget class="QLabel" name="aiLogger">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>CF Nuclear War</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>CURRENTLY CHILLIONG OUT...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="throwGroup">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>310</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>izbaci grupu</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="errorLogger">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>70</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>errorLogger</string>
    </property>
   </widget>
   <widget class="QLabel" name="groupValue">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>100</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>gruopValie : 0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="undoGroup">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>vrati grupu</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>130</y>
      <width>151</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>250</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>820</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="menuBarRestartGame"/>
    <addaction name="actionChoose_cards"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="menuBarRestartGame">
   <property name="text">
    <string>Restart game</string>
   </property>
  </action>
  <action name="actionSelect_theme">
   <property name="text">
    <string>Select theme</string>
   </property>
  </action>
  <action name="actionChoose_cards">
   <property name="text">
    <string>Choose cards</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
